packages/components/modules/comments/CommentItem/index.tsx

```diff
@@ -25,7 +25,6 @@ import useCommentOptions from './useCommentOptions'
 
 const CommentItem: FC<CommentItemProps> = ({
   comment: commentRef,
-  profileId,
```

Comment:
we should also remove the `profileId` from  `CommentItemProps` and from its storybook